src/NuGet.Services.Metadata.Catalog.Monitoring/Providers/ServiceIndexResourceV3Manager.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using System;
+using System.Threading.Tasks;
+using System.Threading;
+using System.Collections.Generic;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class ServiceIndexResourceV3Manager : ResourceProvider, IServiceIndexDiscoveryResource, IServiceIndexHijacker
```

Comment:
>ServiceIndexResourceV3Manager [](start = 17, length = 29)

Resource providers typically have the "Provider" suffix instead of "Manager" right?